tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -0,0 +1,983 @@
+import logging
+import pytest
+import random
+from concurrent.futures import ThreadPoolExecutor
+from time import sleep
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    ignore_leftovers, ManageTest, tier4, tier4a, tier4b, tier4c
+)
+from ocs_ci.ocs import constants, machine, node, ocp
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException, UnexpectedBehaviour
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import ceph_health_check, get_az_count
+from tests import disruption_helpers, helpers
+
+logger = logging.getLogger(__name__)
+
+
+@tier4
+@ignore_leftovers
+class TestRwoPVCFencingUnfencing(ManageTest):
+    """
+    KNIP-677 OCS support for Automated fencing/unfencing RWO PV
+    """
+    pvc_size = 5  # size in Gi
+
+    # Pods of each interface type to be run on nodes which are going to fail
+    num_of_app_pods_per_node = 2
+
+    short_nw_fail_time = 300  # Duration in seconds for short network failure
+
+    prolong_nw_fail_time = 900  # Duration in seconds for prolong network failure
+
+    @pytest.fixture()
+    def setup(
+        self, request, scenario, num_of_nodes, num_of_fail_nodes,
+        disrupt_provisioner, project_factory, multi_pvc_factory, dc_pod_factory
+    ):
+        """
+        Identify the nodes and start DeploymentConfig based app pods using
+        PVC with ReadWriteOnce (RWO) access mode on selected nodes
+        """
```

Comment:
Pleas add some more information in the comment about the parameters (e.g. scenario etc.)

Reply:
done